src/features/game/types/bumpkinItemBuffs.ts

```diff
@@ -79,6 +79,12 @@ export const BUMPKIN_ITEM_BUFF_LABELS: Partial<Record<BumpkinItem, BuffLabel>> =
       boostTypeIcon: powerup,
       boostedItemIcon: CROP_LIFECYCLE.Parsnip.crop,
     },
+    "Sunflower Shield": {
+      shortDescription: translate("bumpkinItemBuff.sunflower.shield.boost"),
+      labelType: "success",
+      boostTypeIcon: powerup,
```

Comment:
```suggestion
      boostTypeIcon: lightning,
```
Just a suggestion, we want to use power up only if it boosts yield of the item. any other effects we use lightning